pkgs/os-specific/linux/lkvm/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv, fetchgit, libbfd, zlib, libaio, glibc }:
+
+stdenv.mkDerivation {
+  name = "lkvm";
+  version = "2020-05-19";
+
+  src = fetchgit {
+    url = "https://git.kernel.org/pub/scm/linux/kernel/git/will/kvmtool.git";
+    rev = "b4fc4f605fc66a0942e88f37b3a4b18671e32b0c";
+    sha256 = "0kyb3rm98wzwgjs06crcgb39547rr4px4vap8ivm7yqnwbphr67f";
+  };
+
+  buildInputs = [ libbfd zlib libaio glibc glibc.static];
+
+  buildPhase = ''
+    make all
```

Comment:
You can use `buildFlags = [ "all" ];` but reading https://git.kernel.org/pub/scm/linux/kernel/git/will/kvmtool.git/tree/INSTALL, the default `buildPhase` should be enough.

Reply:
This does not seem to be resolved. What happens when you remove the `buildPhase` and leave it to the default value?